pints/toy/_constant_model.py

```diff
@@ -0,0 +1,64 @@
+#
+# Constant model with multiple outputs.
+#
+# This file is part of PINTS.
+#  Copyright (c) 2017-2018, University of Oxford.
+#  For licensing information, see the LICENSE file distributed with the PINTS
+#  software package.
+#
+from __future__ import absolute_import, division
+from __future__ import print_function, unicode_literals
+import numpy as np
+import pints
+
+
+class ConstantModel(pints.ForwardModel):
+    """
+    *Extends:* :class:`pints.ForwardModel`.
+    .. math::
+        f(t) &= (a_1,a_2,...,a_k)
+
+    Has a vector of parameters of dimensionality k: each of which
+    is the user-specified output of a given component of the function.
+    This function is mostly useful for unit testing.
+
+    """
+
+    def __init__(self):
+        super(ConstantModel, self).__init__()
+        self._no = None
+        self._parameters = [0]
+
+    def parameters(self):
```

Comment:
What's this method for?

Reply:
It returns the parameters of the model. Happy to remove.

Reply:
Removed!